content/en/docs/contribute/new-content/contributing-content.md

```diff
@@ -0,0 +1,503 @@
+---
+title: Opening a pull request
+slug: new-content
```

Comment:
nit: This file, `Opening a pull request`, is reached by `/docs/contribute/new-content/new-content/`. These different names/phrases will make it harder to understand/associate what is in the page.

Reply:
The next commit will rename this to `open-a-pr.md`.